client/blocks/editor-launch-modal/style.scss

```diff
@@ -0,0 +1,9 @@
+.editor-launch-modal {
+	position: fixed;
+	top: 0;
+	left: 0;
+	width: 100%;
+	height: 100%;
+	background: rgba( 255, 255, 255, 0.7 );
+	z-index: 99999;
```

Comment:
You may want to check if Calypso has a built-in system for z-index order

For reference: [`/client/assets/stylesheets/shared/functions/_z-index.scss`](https://github.com/Automattic/wp-calypso/blob/ce28588835185da73d7e82d0de69f36b4211df9a/client/assets/stylesheets/shared/functions/_z-index.scss)

Not a blocker for this PR